engines/duplicity.go

```diff
@@ -208,24 +208,24 @@ func (d *DuplicityEngine) status() (err error) {
 		return
 	}
 
-	fullBackup := fullBackupRx.FindStringSubmatch(stdout)
+	fullBackup := fullBackupRx.FindAllStringSubmatch(stdout, -1)
 	var fullBackupDate time.Time
-	chainEndTime := chainEndTimeRx.FindStringSubmatch(stdout)
+	chainEndTime := chainEndTimeRx.FindAllStringSubmatch(stdout, -1)
```

Comment:
We might actually want to perform the match inside the `if len(fullBackup) > 0`.